[2024-09-17T13:06:28.171+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-17T13:06:28.219+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: rasa_data_ingestion_dag.create_pet_table manual__2024-09-17T13:06:25.866867+00:00 [queued]>
[2024-09-17T13:06:28.242+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: rasa_data_ingestion_dag.create_pet_table manual__2024-09-17T13:06:25.866867+00:00 [queued]>
[2024-09-17T13:06:28.242+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2024-09-17T13:06:28.272+0000] {taskinstance.py:2888} INFO - Executing <Task(SQLExecuteQueryOperator): create_pet_table> on 2024-09-17 13:06:25.866867+00:00
[2024-09-17T13:06:28.295+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'rasa_data_ingestion_dag', 'create_pet_table', 'manual__2024-09-17T13:06:25.866867+00:00', '--job-id', '18', '--raw', '--subdir', 'DAGS_FOLDER/rasa_data_ingestion_dag.py', '--cfg-path', '/tmp/tmpp1rch398']
[2024-09-17T13:06:28.300+0000] {standard_task_runner.py:105} INFO - Job 18: Subtask create_pet_table
[2024-09-17T13:06:28.305+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=1058) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-09-17T13:06:28.306+0000] {standard_task_runner.py:72} INFO - Started process 1059 to run task
[2024-09-17T13:06:28.391+0000] {task_command.py:467} INFO - Running <TaskInstance: rasa_data_ingestion_dag.create_pet_table manual__2024-09-17T13:06:25.866867+00:00 [running]> on host aca847883b6e
[2024-09-17T13:06:28.599+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Eric Silveira' AIRFLOW_CTX_DAG_ID='rasa_data_ingestion_dag' AIRFLOW_CTX_TASK_ID='create_pet_table' AIRFLOW_CTX_EXECUTION_DATE='2024-09-17T13:06:25.866867+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-09-17T13:06:25.866867+00:00'
[2024-09-17T13:06:28.602+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-17T13:06:28.610+0000] {sql.py:266} INFO - Executing: 
            SELECT column_name
            FROM information_schema.columns
            WHERE table_name = 'events';
          
[2024-09-17T13:06:28.630+0000] {base.py:84} INFO - Retrieving connection 'conn_db_***'
[2024-09-17T13:06:28.665+0000] {base.py:84} INFO - Retrieving connection 'conn_db_***'
[2024-09-17T13:06:28.675+0000] {sql.py:509} INFO - Running statement: 
            SELECT column_name
            FROM information_schema.columns
            WHERE table_name = 'events';
          , parameters: None
[2024-09-17T13:06:28.692+0000] {sql.py:518} INFO - Rows affected: 7
[2024-09-17T13:06:28.746+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-17T13:06:28.747+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=***_data_ingestion_dag, task_id=create_pet_table, run_id=manual__2024-09-17T13:06:25.866867+00:00, execution_date=20240917T130625, start_date=20240917T130628, end_date=20240917T130628
[2024-09-17T13:06:28.805+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-09-17T13:06:28.843+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-09-17T13:06:28.845+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
